dotnet/src/MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders/IJsonStringBuilder.cs

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2022-2023 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+
+namespace MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders
+{
+    public interface IJsonStringBuilder
```

Comment:
This isn't wrong but a delegate is probably a better fit as the C# version of s Java functional interface.